tests/test_SklearnCountVectorizerConverter.py

```diff
@@ -1,15 +1,21 @@
 """
-Tests scikit-learn's binarizer converter.
+Tests scikit-learn's tfidf converter.
```

Comment:
Isn't this count vectorizer converter's test?

Reply:
Fixed.